inc/admin.php

```diff
@@ -754,6 +762,23 @@ function js_templates() {
 		include plugin_dir_path( __FILE__ ) . '../tpl/js-templates.php';
 	}
 
+
+	/**
+	 * Need to render templates for new WP 4.8 widgets when not on the 'widgets' screen or in the customizer.
+	 */
+	function js_widget_templates() {
+		$screen = get_current_screen();
```

Comment:
Aaah, missed this before merging. Oh well, we can look at it in `develop`.

`enqueue_admin_scripts` is called on every page, but it has the `self::is_admin()` check to make sure the current page is using Page Builder (unless `$force` is true).

https://github.com/siteorigin/siteorigin-panels/blob/develop/inc/admin.php#L105

Right now `js_widget_templates` is being called on pages like WooCommerce/EDD product pages and any other custom post type. This opens us up to potential conflicts.

Plus, having all this JS widget code in `enqueue_admin_scripts` gives us the benefit of having a single function we can call to enqueue everything to make Page Builder work for situations where it's outside the context of a page. For example, the custom home page interface.